packtools/sps/models/sps_package.py

```diff
@@ -980,16 +980,6 @@ def create_alternatives(node, assets_data):
         parent.replace(node, alternative_node)
 
 
-def formatted_text(any_article_title_node):
-    if any_article_title_node is None:
-        return
-    node = deepcopy(any_article_title_node)
-    for xref in node.findall(".//xref"):
```

Comment:
`xref` é uma tag de SPS e não uma tag genérica...

Reply:
Sugere que o que for específico de SPS eu deixe em sps_package.py (para que o utilitário xml_utils não seja desvirtuado)?

Reply:
@rafaelpezzuto pode criar um módulo intermediário